www/src/pages/foundations/colors.tsx

```diff
@@ -130,123 +129,46 @@ export default function ColorsPage({ data }: IColorsPage) {
         {/* eslint-disable-next-line react/jsx-pascal-case */}
         <SEO title="Colors" />
         <h1>Colors</h1>
-        <div className="d-flex flex-wrap">
+        <div className="color-palette mb-3">
           {colors
             .slice(0, 3)
             .map(({ themeName, unusedLevels }) => renderColorRamp(themeName, unusedLevels))}
-          <div
-            style={{
-              flexBasis: '19%',
-              marginRight: '1%',
-              marginBottom: '2rem',
-            }}
-          >
+          <div>
             <h2 className="h5">accents</h2>
 
-            <Swatch name="$accent-a" colorClassName="bg-accent-a" />
-            <Swatch name="$accent-b" colorClassName="bg-accent-b" />
+            <Swatch name="var(--pgn-color-accent-a)" colorClassName="bg-accent-a" />
+            <Swatch name="var(--pgn-color-accent-b)" colorClassName="bg-accent-b" />
           </div>
 
           {colors
             .slice(3)
             .map(({ themeName, unusedLevels }) => renderColorRamp(themeName, unusedLevels))}
         </div>
 
-        <h3>SCSS Color Usage</h3>
-        <p>Include these colors in scss files in one of two ways:</p>
+        <h3>CSS Color Usage</h3>
 
         <h4>Variable name</h4>
         <code className="d-block mb-4 lead bg-gray-100 p-3">
-          {'// $color_name-level '}
+          {'// var(--pgn-color-name-level) '}
           <br />
-          $primary-100
+          var(--pgn-color-primary-100)
           <br />
-          $primary-200
+          var(--pgn-color-primary-200)
           <br />
-          $brand-100
+          var(--pgn-color-brand-100)
           <br />
-          $brand-200
-        </code>
-
-        <h4>Mixin (deprecated)</h4>
-        <code className="d-block mb-4 lead bg-gray-100 p-3">
-          theme-color($color-name, $variant)
+          var(--pgn-color-brand-200)
         </code>
 
+        <h4>With default value</h4>
         <p>
-          Using the variable name instead of the theme-color mixin will make
-          later upgrade paths easier. Paragon may begin to adopt CSS variables
-          for theming and attempt to eliminate mixins from the public api.
+          Using a default value in CSS variables allows you to set a default value for a variable,
```

Comment:
I would remove "you" here.
```
By refraining from using second-person pronouns, such as "you," the documentation can be written in a more objective and inclusive manner. It allows the readers to feel that the information provided is applicable to anyone who is accessing the documentation, rather than singling out specific individuals.

Instead, it is recommended to use more general and inclusive language, such as imperative statements or noun phrases, to convey instructions or describe concepts. This approach helps maintain a professional and consistent tone throughout the documentation while accommodating various users.
```

Reply:
Thanks 👍